website/docs/sdk-reference/dotnet.md

```diff
@@ -46,9 +53,11 @@ else
 ```
 
 ### 5. Dispose *ConfigCat* client
-You can safely `Dispose()` the client instance and release all associated resources on application exit.
+You can safely shut down all clients at once or individually and release all associated resources on application exit.
```

Comment:
Why not `dispose`?

Reply:
Probably I copied this text over from Dart's docs. I'll change it to 'dispose'.